src/graphql/objects/state.ts

```diff
@@ -0,0 +1,20 @@
+import { Field, ObjectType } from "type-graphql";
+import { Epochs } from ".";
+
+@ObjectType({ simpleResolvers: true })
+export class State {
+  @Field()
+  blockId?: string;
+
+  @Field()
+  height?: number;
+
+  @Field()
+  maxBoxGlobalIndex?: bigint;
+
+  @Field()
+  maxTransactionGlobalIndex?: bigint;
```

Comment:
I suggested such naming, however I think we can make it more semantic by renaming it to just `boxGlobalIndex` and `transactionGlobalIndex`. Can you please change it?